extensions/qcom/GLSL_QCOM_image_processing2.txt

```diff
@@ -0,0 +1,212 @@
+    |       uvec2     refCoord,            | left corner of the block in target and      |
+    |       uvec2     blockSize)           | reference images. The error metric is the   |
+    |                                      | sum of the squares of the values (SSD).     |
+    +--------------------------------------+---------------------------------------------+
+    | vec4 textureBlockMatchWindowSADQCOM( | Windowed block matching operation that      |
+    |       sampler2D target               | measures the correlation (or similarity)    |
+    |       uvec2     targetCoord,         | of the target block and reference block.    |
+    |       sampler2D reference,           | targetCoord and refCoord specify the bottom-|
+    |       uvec2     refCoord,            | left corner of the block in target and      |
+    |       uvec2     blockSize)           | reference images. The error metric is the   |
+    |                                      | sum of the absolute values (SAD).           |
+    +--------------------------------------+---------------------------------------------+
+    | vec4 textureBlockMatchGatherSSDQCOM( | Gathered block matching operation that      |
+    |       sampler2D target               | measures the correlation (or similarity)    |
+    |       uvec2     targetCoord,         | of the target block and reference block.    |
+    |       sampler2D reference,           | targetCoord and refCoord specify the bottom-|
+    |       uvec2     refCoord,            | left corner of the block in target and      |
+    |       uvec2     blockSize)           | reference images. The error metric is the   |
+    |                                      | sum of the squares of the values (SSD).     |
+    +--------------------------------------+---------------------------------------------+
+    | vec4 textureBlockMatchGatherSADQCOM( | Gathered block matching operation that      |
+    |       sampler2D target               | measures the correlation (or similarity)    |
+    |       uvec2     targetCoord,         | of the target block and reference block.    |
+    |       sampler2D reference,           | targetCoord and refCoord specify the bottom-|
+    |       uvec2     refCoord,            | left corner of the block in target and      |
+    |       uvec2     blockSize)           | reference images. The error metric is the   |
+    |                                      | sum of the absolute values (SAD).           |
+    +--------------------------------------+---------------------------------------------+
+
+    * target and reference specify the two input textures. If
+      the block from one texture is going to be used repeatedly across multiple
+      instructions, that texture should be specified as the referenceTexture.
+
+    * targetCoord and referenceCoord specify the texture coordinates (in texels)
+      of the bottom left corner of the block.
```

Comment:
This mostly repeats the description in the table. We should remove it from one or other place. (And also, a nit, these descriptions normally go before the table of functions, rather than after)

Reply:
I updated the texts in the table, and removed it.